includes/Integrations/Events_Calendar/Compatibility/TEC_V6_Compatibility.php

```
@@ -0,0 +1,773 @@
+
+            if ( empty( $tec_data['_EventTimezone'] ) ) {
+                $tec_data['_EventTimezone'] = wp_timezone_string();
+            }
+
+            if ( ! isset( $tec_data['_EventAllDay'] ) ) {
+                $tec_data['_EventAllDay'] = 'no';
+            }
+        }
+
+        return $tec_data;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'convert_to_tec_format'. (Unused Code Rules)

(UnusedPrivateMethod)